gitbootcamp.rst

```diff
@@ -107,7 +107,7 @@ Create a new branch and switch to it::
 
 This is equivalent to::
 
-   # create a new branch off 'master', without checking it out
+   # create a new branch off master, without checking it out
    git branch <branch-name> master
    # check out the branch
```

Comment:
This suggestion is along the same lines of making wording more explicit. Also, the usage of "checkout" is more appropriate here than "check out", and more closely matches the Git command.
```suggestion
   # checkout the newly created branch
```

Reply:
I disagree here, I think that comments should not necessarily follow the command names but rather be correct English, so the infinitive "check out" or "switch to".

Reply:
Generally speaking, code comments have a tendency to be significantly less restricted by proper English. However, both "checkout" and "check out" are probably acceptable in this case.